x/ibchooks/marker_hooks.go

```diff
@@ -96,6 +96,7 @@ func (h MarkerHooks) createNewIbcMarker(ctx sdktypes.Context, data transfertypes
 		allowForceTransfer,
 		[]string{},
 	)
+	h.adjustSupply(ctx, marker)
```

Comment:
the terminology used here is confusing ... it would be more clear if this was structured as "get existing supply" then add the incoming amount to it.  Existing supply would be zero or more...

Reply:
Thanks, I was waffling on what to call it that described the action better.